src/commands/index.ts

```diff
@@ -1,10 +1,10 @@
-import { ICommand } from './command.interface';
+import { Command } from './command';
 import { PingCommand } from './ping.command';
 import { DefaultCommand } from '@/commands/default.command';
 
 const defaultCommand = new DefaultCommand();
 
-const commands: ICommand[] = [
+const commands: Command[] = [
     defaultCommand,
     new PingCommand()
```

Comment:
Maybe for consistency's sake - extract `new PingCommand()` to constant as well?

Reply:
No need. Only reason I extracted that one to a constant is because I import it and use it elsewhere explicitly, and I didn't want to keep using singleton references instead of multiple instances.

Essentially, the only reason the `commands` array exists is because there's no way to iterate over the `exports` in TS (that I'm aware of). Since `new PingCommand()` is only used when we're iterating over these properties, and doesn't need to be found explicitly, there's no need for it. Whereas `defaultCommand` is used in `message.handler.ts` where I do `?? defaultCommand` or something.